client/src/services/api/hooks/codeReview/timeMetricsUtils.ts

```diff
@@ -0,0 +1,91 @@
+import { VOTES } from "src/constants/codeReviewMetrics.constants";
+import { getTimeInSeconds } from "src/utils/dateUtil";
+
+import { IFetchedPullRequestVotesTimeline } from "./codeReviewTypes";
+
+export class TimeMetrics {
+  static getFirstReviewResponseTime(votesHistoryTimeline: IFetchedPullRequestVotesTimeline[]): number | null {
+    if (!votesHistoryTimeline.length) {
+      return null;
+    }
+
+    const [firstVote] = votesHistoryTimeline;
+    const { timeOfVote, reviewerAddedTime } = firstVote;
+
+    return timeOfVote && reviewerAddedTime ? getTimeInSeconds(timeOfVote, reviewerAddedTime) : null;
+  }
+
+  static getPullRequestApprovalTime(
+    creationDate: string,
+    reviewers: IFetchedPullRequestVotesTimeline[],
+    votesHistoryTimeline: IFetchedPullRequestVotesTimeline[],
+    selectedReviewers: Map<string, boolean>,
+  ) {
+    const isReviewersSelected = selectedReviewers.size > 0;
```

Comment:
if(selectedReviewers.size) -> can we use shorthand since `0` will be `falsy` value